src/components/TitleCardList.tsx

```diff
@@ -152,52 +40,169 @@ const TitleCardList: React.FC<IProps> = ({
   icon,
   isCode,
 }) => (
-  <Table className={className}>
-    <TableHeader>
-      {icon && <StyledIcon name={icon} />}
+  <Box
+    bg="background"
+    boxShadow="tableBox"
+    width="100%"
+    margin={8}
+    border-radius="sm"
+    className={className}
+  >
+    <Flex
+      bg="ednBackground"
+      align="center"
+      justifyContent="space-between"
+      padding={4}
+      flexDirection="row-reverse"
+      fontWeight="600"
+      borderBottom={0.25}
+      borderBottomStyle="solid"
+      borderBottomColor="text"
+    >
+      {icon && (
+        <ChakraIcon
+          as={Icon}
+          name={icon}
+          _hover={{
+            path: { fill: "tranparent" },
+          }}
+        />
+      )}
       <Translation id={headerKey} />
       {isCode && (
-        <CodeBoxHeader>
-          <Red />
-          <Yellow />
-          <Green />
-        </CodeBoxHeader>
+        <Hide below="s">
+          <Flex>
+            <Box
+              width="12px"
+              height="12px"
+              bg="fail300"
+              marginRight={2}
+              borderRadius="full"
+            />
+            <Box
+              width="12px"
+              height="12px"
+              bg="gridYellow"
+              marginRight={2}
+              borderRadius="full"
+            />
+            <Box
+              width="12px"
+              height="12px"
+              bg="success300"
+              marginRight={2}
+              borderRadius="full"
+            />
+          </Flex>
+        </Hide>
       )}
-    </TableHeader>
+    </Flex>
     {content.map((listItem, idx) => {
       const { title, description, caption, link, image, alt, id } = listItem
       const isLink = !!link
+
       return isLink ? (
-        <ItemLink key={id || idx} to={link}>
-          {image && <Image image={image} alt={alt || ""} />}
-          <LeftContainer>
-            <ItemTitle>{title}</ItemTitle>
+        <LinkBox
+          key={id || idx}
+          as={Link}
+          href={link}
```

Comment:
`LinkBox` is not being used as intended here because it needs its partner `LinkOverlay` which would be the one to receive `as={Link}` `hideArrow`, and `to={link}` (using `to` instead of `href`

See the [Chakra docs](https://chakra-ui.com/docs/components/link-overlay/usage)

Maybe the most natural place for `LinkOverlay` is to wrap it around the `title` content in place of `Box` which is being given no styles anyway. See the comment farther down at [line 136](https://github.com/ethereum/ethereum-org-website/pull/8067/files#diff-0f98fcf4d26b7b815c20d7c30103440e8ebba1dd06ef71932ef0fccd7f7bcc60R136)